packages/experimental/Menu/src/MenuList/MenuList.types.ts

```diff
@@ -1,12 +1,20 @@
 import type { IViewProps } from '@fluentui-react-native/adapters';
+import { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
 import { IBackgroundColorTokens, LayoutTokens } from '@fluentui-react-native/tokens';
 
 export const menuListName = 'MenuList';
 
 export interface MenuListTokens extends LayoutTokens, IBackgroundColorTokens {}
 
 export interface MenuListProps extends Omit<IViewProps, 'onPress'> {
+  checked?: Record<string, boolean>;
```

Comment:
checked/defaultChecked state is supplied as mapping of names to a boolean. The boolean could be considered redundant, but makes the lookup for checked state easier/faster. Alternative shape for API could be to take an array of names of controls instead.